utils/training_loop.py

```diff
@@ -13,42 +13,47 @@ def training_loop(experiment, env, agent, model):
         print('Only Inference')
```

Comment:
Add documentation explaining the function 'training_loop'.

Reply:
Done